ray_julia_jll/build/build_tarballs.jl

```diff
@@ -19,13 +19,13 @@ if abspath(PROGRAM_FILE) == @__FILE__
     isdir(TARBALL_DIR) || mkdir(TARBALL_DIR)
 
     @info "Building ray_julia_jll..."
-    run(`$(Base.julia_cmd()) --project=$JLL_PATH $JLL_PATH/deps/build_jll.jl `)
+    include("build.jl")
```

Comment:
are we sure that we don't need to fork a new julia process to run this build step?  why were we doing that in the first place?

Reply:
The reason we previously did this was due to the `build_jll.jl` and the the `make.jl` having separate projects with different dependencies.